packages/manager/modules/pci/src/projects/project/storages/volume-backup/list/delete/template.html

```diff
@@ -0,0 +1,38 @@
+<form
+    id="volumeBackupDelete"
+    name="volumeBackupDelete"
+    data-ng-submit="$ctrl.onDeleteVolumeBackupClick()"
+    novalidate
+>
+    <oui-modal
+        data-heading="{{:: 'pci_projects_project_storages_volume_backup_list_delete_title' | translate }}"
+        data-primary-label="{{:: 'pci_projects_project_storages_volume_backup_list_delete_action_confirm' | translate }}"
+        data-secondary-label="{{:: 'pci_projects_project_storages_volume_backup_list_delete_action_cancel' | translate }}"
+        data-primary-disabled="$ctrl.isDeleting || volumeBackupDelete.$invalid"
+        data-secondary-action="$ctrl.onDeleteVolumeBackupCancelClick()"
+        data-on-dismiss="$ctrl.goBack()"
+        data-loading="$ctrl.isDeleting"
+        data-type="warning"
+    >
```

Comment:
**nitpick:** Use `ng-form` and remove the native form element
```suggestion
    <oui-modal
        data-ng-form="volumeBackupDelete"
        data-heading="{{:: 'pci_projects_project_storages_volume_backup_list_delete_title' | translate }}"
        data-primary-label="{{:: 'pci_projects_project_storages_volume_backup_list_delete_action_confirm' | translate }}"
        data-secondary-label="{{:: 'pci_projects_project_storages_volume_backup_list_delete_action_cancel' | translate }}"
        data-primary-disabled="$ctrl.isDeleting || volumeBackupDelete.$invalid"
        data-secondary-action="$ctrl.onDeleteVolumeBackupCancelClick()"
        data-on-dismiss="$ctrl.goBack()"
        data-loading="$ctrl.isDeleting"
        data-type="warning"
    >
```

Reply:
yes it's more relevant in this case.

Reply:
done